packages/extensions/src/node/utils/get-extensions.ts

```diff
@@ -51,7 +53,17 @@ export async function resolvePackageExtensions(root: string, extensionNames?: st
 					app: extensionOptions.path.app,
 					api: extensionOptions.path.api,
 				},
-				entries: extensionOptions.entries.map((entry) => pick(entry, 'name', 'type')),
+				entries: extensionOptions.entries
+					.filter((entry) => {
+						const isUnique = loadedEntryNames.includes(entry.name) === false;
+
+						if (isUnique) {
+							loadedEntryNames.push(entry.name);
+						}
+
+						return isUnique;
+					})
+					.map((entry) => pick(entry, 'name', 'type')),
```

Comment:
Similar to my first comment: Let's move `entries` out of this `push` and use a for loop.